### ISSUE TITLE
Error in FindIntegrationAnchors

### ISSUE THREAD

**Issue by JLiLab** [satijalab/seurat#979]
Hi, 

I was running the tutorial using the human pancreatic islet cell datasets, and got the following error message in FindIntegrationAnchors: 

```R
pancreas.anchors <- FindIntegrationAnchors(object.list = reference.list, dims = 1:30)

Computing 2000 integration features
Scaling features for provided objects
   |++++++++++++++++++++++++++++++++++++++++++++++++++| 100% elapsed = 04s
Finding all pairwise anchors
   |                                                  | 0 % ~calculating  Running CCA
Error in irlba(A = mat3, nv = num.cc) : 
  BLAS/LAPACK routine 'DLASCL' gave error code -4
```
<details><summary>Here is my session information: </summary>
R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.6

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] Seurat_3.0.0.9000

loaded via a namespace (and not attached):
 [1] httr_1.3.1          tidyr_0.8.2         viridisLite_0.3.0   jsonlite_1.5       
 [5] splines_3.5.1       lsei_1.2-0          R.utils_2.7.0       gtools_3.8.1       
 [9] Rdpack_0.10-1       assertthat_0.2.0    yaml_2.2.0          ggrepel_0.8.0      
[13] globals_0.12.4      pillar_1.3.0        lattice_0.20-35     reticulate_1.10    
[17] glue_1.3.0          digest_0.6.18       RColorBrewer_1.1-2  SDMTools_1.1-221   
[21] colorspace_1.3-2    cowplot_0.9.3       htmltools_0.3.6     Matrix_1.2-15      
[25] R.oo_1.22.0         plyr_1.8.4          pkgconfig_2.0.2     bibtex_0.4.2       
[29] tsne_0.1-3          listenv_0.7.0       purrr_0.2.5         scales_1.0.0       
[33] RANN_2.6            gdata_2.18.0        Rtsne_0.14          tibble_1.4.2       
[37] ggplot2_3.1.0       ROCR_1.0-7          pbapply_1.3-4       lazyeval_0.2.1     
[41] survival_2.43-1     magrittr_1.5        crayon_1.3.4        R.methodsS3_1.7.1  
[45] future_1.10.0       MASS_7.3-51.1       gplots_3.0.1        ica_1.0-2          
[49] tools_3.5.1         fitdistrplus_1.0-11 data.table_1.11.8   gbRd_0.4-11        
[53] stringr_1.3.1       plotly_4.8.0        munsell_0.5.0       cluster_2.0.7-1    
[57] irlba_2.3.2         bindrcpp_0.2.2      compiler_3.5.1      rsvd_1.0.0         
[61] caTools_1.17.1.1    rlang_0.3.0.1       grid_3.5.1          ggridges_0.5.1     
[65] rstudioapi_0.8      htmlwidgets_1.3     igraph_1.2.2        bitops_1.0-6       
[69] npsurv_0.4-0        gtable_0.2.0        codetools_0.2-15    R6_2.3.0           
[73] zoo_1.8-4           dplyr_0.7.8         future.apply_1.0.1  bindr_0.1.1        
[77] KernSmooth_2.23-15  metap_1.0           stringi_1.2.4       parallel_3.5.1     
[81] Rcpp_1.0.0          png_0.1-7           tidyselect_0.2.5    lmtest_0.9-36   
</details>
I am not sure what is the problem.  Any suggestion?

James

**Comment by andrewwbutler** [comment]
Hi James,

Thanks for reporting. This should be fixed as of e2c5d4f. Please try reinstalling.

**Comment by JLiLab** [comment]
Problem solved. Thank you!
